ILRepack/RepackOptions.cs

```diff
@@ -210,8 +212,8 @@ void Parse()
         /// </summary>
         internal void Validate()
         {
-            if (string.IsNullOrEmpty(KeyFile) && DelaySign)
```

Comment:
`KeyFile == null` is the check used everywhere else in the source, so when I added `KeyContainer == null` I made `KeyFile` consistent.